pkg/app/context.go

```diff
@@ -17,6 +17,15 @@ func NewContext() Context {
 func (ctx Context) SyncReposOnce(st *state.HelmState, helm state.RepoUpdater) []error {
 	var errs []error
 
+	hasInstalled := false
```

Comment:
This should be set to `true` on init. Otherwise L22 never turns true

Reply:
...Which seems to be the case of the integration test failure

Reply:
Yes I made a mistake, I invert the condition